core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -298,23 +299,30 @@ angular.module('oppia').component('contributionsAndReview', {
           ctrl.activeSuggestionType === suggestionType);
       };
 
+      const openQuestionSuggestionModal = function(
+          suggestionId, question = undefined) {
+        var suggestion = ctrl.contributions[suggestionId].suggestion;
+        var reviewable = ctrl.activeTabType === ctrl.TAB_TYPE_REVIEWS;
+        var contributionDetails = ctrl.contributions[suggestionId].details;
+        var skillId = suggestion.change.skill_id;
+        ContextService.setCustomEntityContext(
+          IMAGE_CONTEXT.QUESTION_SUGGESTIONS, skillId);
+        SkillBackendApiService.fetchSkillAsync(skillId).then((skillDict) => {
+          var misconceptionsBySkill = {};
+          var skill = skillDict.skill;
+          misconceptionsBySkill[skill.getId()] = skill.getMisconceptions();
+          _showQuestionSuggestionModal(
+            suggestion, contributionDetails, reviewable,
+            misconceptionsBySkill, question);
+          $rootScope.$apply();
+        });
+      };
+
       ctrl.onClickViewSuggestion = function(suggestionId) {
         var suggestion = ctrl.contributions[suggestionId].suggestion;
         var reviewable = ctrl.activeTabType === ctrl.TAB_TYPE_REVIEWS;
```

Comment:
Can we pass these into `openQuestionSuggestionModal()`?

Reply:
Addressed in https://github.com/oppia/oppia/pull/15303. Thanks!